tests/e2e/vsphere_volume_expansion.go

```diff
@@ -733,14 +733,11 @@ var _ = ginkgo.Describe("Volume Expansion Test", func() {
 		if !featureEnabled {
 			ginkgo.By("File system resize should not succeed since SPS service is down" +
 				" and cns new sync feature is disabled. Expecting an error")
-			if guestCluster {
-				expectedErrMsg = "didn't find a plugin capable of expanding the volume"
-			} else {
-				expectedErrMsg = "failed to expand volume"
-			}
+			expectedErrMsg = "VolumeResizeFailed"
 			framework.Logf("Expected failure message: %+q", expectedErrMsg)
-			err = waitForEvent(ctx, client, namespace, expectedErrMsg, pvclaim.Name)
+			isFailureFound, err := getEventsListAndVerifyPvcError(client, namespace, pvclaim, expectedErrMsg)
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			gomega.Expect(isFailureFound).To(gomega.BeTrue(), expectedErrMsg)
```

Comment:
change `expectedErrMsg` to `"Expected error %v to occur but did not occur", expectedErrMsg`